tests/functional/queries_with_sql_types.py

```diff
@@ -12,20 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import psycopg2 as pg
 import pytest
 
+from psycopg2._psycopg import connection, cursor
+from psycopg2 import connect, Error
+
+# all imports from errors are OK if you can find such exception class in docs
```

Comment:
Could you add link to docs so others can navigate without googling?

Reply:
done